quantum/action.c

```diff
@@ -1183,6 +1183,15 @@ bool is_tap_action(action_t action) {
     return false;
 }
 
+uint16_t get_tap_keycode(uint16_t keycode) {
```

Comment:
Technically, there is `SH_T(kc)`, as well (swap on hold, keycode on tap)

Reply:
Added `SH_T(kc)`.